frontend/src/metabase/lib/entities.js

```diff
@@ -32,6 +32,8 @@ import type { APIMethod } from "metabase/lib/api";
 
 type EntityName = string;
 
+type EntityForm = any;
```

Comment:
I'd also use the `FormDef` type from `metabase/containers/Form.jsx`

```
import type { FormDef } from "metabase/containers/Form";
```

Like `FormFieldDef` above you'll need to 

```
export type FormDef = {
```

in `metabase/containers/Form.jsx`